src/main/java/seedu/momentum/model/project/Project.java

```diff
@@ -90,6 +94,10 @@ public Deadline getDeadline() {
         return deadline;
     }
 
+    public boolean hasDeadline() {
+        return !deadline.isEmpty();
+    }
+
```

Comment:
Similar to above, should this be in deadline instead?

Reply:
I feel that it is ok to have this method as well as the hasDescription() method in the Project class, as a Project should be aware of the contents of its own fields and I can directly ask a Project whether it possesses these fields.

However, what you sounds sounds fine to me too. What do the others think?

Reply:
I'd prefer keeping it in deadline like what Farrell suggested

Reply:
I agree with Farrell

Reply:
Yup, I think there is no need to put this method under Project. We can just keep it in Deadline!